### PR TITLE
Bugfix - persist conditional reveal values

### PR DESCRIPTION
**Pull request by joefitter**
* updated form to persist conditional reveal values
* dont validate field if conditional reveal is set to false

### CODE REVIEW

File: pages/common/routers/form.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ module.exports = ({
     });
     req.form.values = mapValues(req.form.values, trim);
     req.form.values = cleanModel(req.form.values);
+    Object.assign(req.form.values, { ...pickBy(req.body, (field, key) => key.includes('conditional-reveal')) });
```
Comment (lennym) [inline pages/common/routers/form.js]: I don't think this is the right way to go about this. It's automatically whitelisting any field with `conditional-reveal`. 

I'd iterate over the schema, and do something like this to get the keys of the associated radio button.

```js
Object.keys(schema).filter(key => schema[key].conditionalReveal).map(key => `conditional-reveal-${key}`)
```

i.e. specifically find the fields which are conditional first, and then map them to their corresponding keys.

Comment (joefitter) [reply pages/common/routers/form.js]: FINE

Comment (lennym) [reply pages/common/routers/form.js]: 😢

Comment (joefitter) [reply pages/common/routers/form.js]: 🤗

**Pull request merged**